app/install/ubuntu.md

```diff
@@ -27,7 +27,7 @@ Start by downloading the corresponding package for your configuration:
 
     ```bash
     $ sudo apt-get update
-    $ sudo apt-get install netcat lua5.1 openssl libpcre3 dnsmasq
```

Comment:
We have to make sure that the OpenResty bundle puts LuaJIT in the $PATH of the system it is installed on, for all ours distributions.